config/game_options.txt

```diff
@@ -141,6 +141,18 @@ MIDROUND_ANTAG BLOB
 #MIDROUND_ANTAG  RAGINMAGES
 #MIDROUND_ANTAG  MONKEY
 
+##A time in minutes a round must progress before a shuttle call can be successful without anything being very wrong with the station.
+SHUTTLE_BOREDOM_CHECK 60
+
+##A ratio (living crew/all crew) used to determine if mass crewmember death is a valid reason to call the shuttle
+SHUTTLE_LIFE_CHECK 0.7
+
+##A ratio (living antag crew/all living crew) used to determine if mass antags is a valid reason to call the shuttle
+SHUTTLE_ANTAG_OVERRUN 0.5
+
+##A ratio (current station/initial station) used to determine if a badly damaged station is a valid reason to call the shuttle
+SHUTTLE_INFRASTRUCTURE_CHECK 0.7
+
```

Comment:
The defaults are still in place, but it'd be better to put them in writing so the headmins can quibble ~in~effectively about it.